example.yaml

```diff
@@ -0,0 +1,216 @@
+hero:
```

Comment:
Consider this shorter format:

```yaml
player:
  - hero:
      hp: [30, 7]
      str: [12, 3]
      spd: [11, 2]
enemy:
  - rat:
      type: common
      hp: [10, 3]
      str: [5, 2] 
      spd: [16, 2]
  - wolf:
      type: common
      hp: [15, 3]
      strength: [8, 2]
      speed: [12, 2]
  - skeleton:
      type: rare
      hp: [30, 3]
      str: [10, 2]
      spd: [10, 2]
  - balrog:
      type: legendary
      hp: [200, 3]
      str: [200, 2]
      spd: [14, 2]
```

In addition to being shorter, you can keep the shorter Stat struct

Reply:
yes, i'm working on this :)

Reply:
done!